GPU/D3D11/DepalettizeShaderD3D11.cpp

```diff
@@ -103,10 +122,17 @@ ID3D11ShaderResourceView *DepalShaderCacheD3D11::GetClutTexture(GEPaletteFormat
 	data.pSysMem = rawClut;
 	// Regardless of format, the CLUT should always be 1024 bytes.
 	data.SysMemPitch = 1024;
+
+	DepalTextureD3D11 *tex = new DepalTextureD3D11();
+	// TODO: Look into 1D textures
 	ASSERT_SUCCESS(device_->CreateTexture2D(&desc, &data, &tex->texture));
 	ASSERT_SUCCESS(device_->CreateShaderResourceView(tex->texture, nullptr, &tex->view));
 	tex->lastFrame = gpuStats.numFlips;
 	texCache_[realClutID] = tex;
+
+	if (expandTo32bit) {
+		delete[] expanded;
```

Comment:
Well, since it's nullptr otherwise, this could be unconditional, not that it matters.

-[Unknown]